src/components/CourseEnrollments/index.jsx

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
-import { withRouter } from 'react-router';
 import PropTypes from 'prop-types';
-import qs from 'query-string';
 import moment from 'moment';
-import { omitBy, isNaN } from 'lodash';
-import { Table, StatusAlert, Pagination } from '@edx/paragon';
+import qs from 'query-string';
+import { withRouter } from 'react-router';
+import { Pagination, Icon, StatusAlert, Table } from '@edx/paragon';
+import { isNaN, omitBy } from 'lodash';
 
+import 'font-awesome/css/font-awesome.css';
 import './CourseEnrollments.scss';
 
-const StatusMessage = props => (
+const StatusAlertWrapper = props => (
```

Comment:
Recommend pulling this out into a new file - this file is already getting quite long and this wrapper + the proptype definition for it could be pulled used elsewhere